auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.java

```diff
@@ -64,27 +66,43 @@ public void saveCredentials(@NonNull Credentials credentials) {
      * @param callback the callback that will receive a valid {@link Credentials} or the {@link CredentialsManagerException}.
      */
     public void getCredentials(@NonNull final BaseCallback<Credentials, CredentialsManagerException> callback) {
+        getCredentials(null, callback);
+    }
+
+    /**
+     * Retrieves the credentials from the storage and refresh them if they have already expired.
+     * It will fail with {@link CredentialsManagerException} if the saved access_token or id_token is null,
+     * or if the tokens have already expired and the refresh_token is null.
+     *
+     * @param callback the callback that will receive a valid {@link Credentials} or the {@link CredentialsManagerException}.
+     */
+    public void getCredentials(@Nullable final String scope,
```

Comment:
I think I'd rather mark it as `@NonNull` and ignore the lint warning on the call above, this way users don't call this method passing null on purpose when the other method is available. Thoughts on this approach?

Reply:
I'll switch this function to have a `@NonNull` scope and then create a helper function that takes in a nullable scope.